specification/2.0/README.md

```diff
@@ -792,6 +792,8 @@ Valid accessor type and component type for each attribute semantic property are
 
 `POSITION` accessor **must** have `min` and `max` properties defined.
 
+All Morph Target's accessors **must** have the same `count` which must match the `count` of accessors of the original mesh.
```

Comment:
You use `must` twice for the same directive.  How about something like this?

> The `count` of each Morph Target accessor **must** exactly match the `count` of the accessor for the original mesh.

Reply:
> The `count` of each Morph Target accessor **must** exactly match the `count` of the accessor for the original mesh.

Is it clear what "the accessor" means?
We're trying to say that all vertex data accessors have the same `count` - no matter original mesh or morph target.

Reply:
How about:

> All Morph Target's accessors **must** have the same `count` as the accessors of the original mesh.

or

> For a given mesh, all vertex data accessors, including morph target accessors, **must** have the same `count`.

Reply:
or replace "mesh" with "primitive"